blaze/utils.py

```diff
@@ -310,3 +310,13 @@ def __get__(self, instance, owner):
             return self
 
         return self._f(instance)
+
+
+class BlazeDeprecationWarning(DeprecationWarning):
```

Comment:
is there a reason to make these classes rather than just using the superclass directly?